### DISCUSSION TITLE
Move to streetcomplete organization?

### DISCUSSION THREAD

**Discussion by FloEdelmann** [Helium314/SCEE#588] [General]
Sometimes, issues are opened in SCEE that should rather be in SC or vice-versa (e.g. https://github.com/streetcomplete/StreetComplete/issues/5775). Currently, we have to close the issue and comment that it should rather go in the other repository.

I suggest to move the [SCEE repository](https://github.com/Helium314/SCEE) to the [streetcomplete organization](https://github.com/streetcomplete), so that transferring issues between the repositories would be supported by GitHub. After moving the repo, `github.com/Helium314/SCEE` would automatically be redirected to `github.com/streetcomplete/SCEE`, so no links would be broken.

However, I understand that maybe either @Helium314 would like to keep this repo as personal, or that @westnordost would not like to "promote" SCEE to that "official" status, so feel free to reject my suggestion.

**Comment by mnalis** [comment]
I'm not either of the project owners (so if they'd both be happier with that merging, far it be from me to object!), but I don't think that  the benefit of occasional easier _"transfer of misplaced issue"_ would warrant asking for such (big) organizational change... 

_(Now, if we had **dozens** of such issues misopened **every week**, it perhaps might sway the scales, but I'm not seeing the problem that needs solving currently...)_

**Comment by Helium314** [comment]
I would be ok with this change, or also with leaving it as is.
Though I'm a little afraid about the semi-official status... SCEE often has lower quality than SC (e.g. lazy hacks, or rather badly designed UI components,...), so it doesn't "feel right" to have this sort of tighter connection.

**Comment by westnordost** [comment]
I also don't fancy it that much. Basically for everything under the StreetComplete organization, I feel responsible.

**Reply by Helium314** [reply]
I don't want you to feel responsible for SCEE in any way, also I don't want other people complaining to you about SCEE stuff. So let's keep it as it is currently.
